funs/database_fun.R

```diff
@@ -67,46 +117,80 @@ download_project <- function(project, Tstart, Tend) {
         as_tibble()
 
     log_debug(paste("Got", nrow(stations), "stations for project", project))
-    apply(stations, 1, FUN  = insert_location_info_vectorized, conn = pool)
+    apply(stations, 1, FUN  = insert_location_info_vectorized, conn = conn)
 
```

Comment:
Why are you using v1 in line 123, while in 120 you use just the code? What is the difference?

Reply:
Both functions don't return anything, so v1 is obsolete, fixed in https://github.com/rivm-syso/Analyse-Together/pull/75/commits/b5888556f88d07de23fad074c2ccadea42bd60e5